<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formpelanggan</class>
 <widget class="QWidget" name="Formpelanggan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="pelangganLabel">
      <property name="text">
       <string>Id pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idpelangganLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="namaPelangganLabel">
      <property name="text">
       <string>Nama Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="namaPelangganLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="alamatPelangganLabel">
      <property name="text">
       <string>Alamat Pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="alamatPelangganLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="kodePosLabel">
      <property name="text">
       <string>Kode Pos</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="kodePosLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="telponLabel">
      <property name="text">
       <string>Telpon</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="telponLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="tanggalDaftarLabel">
      <property name="text">
       <string>Tanggal Daftar</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateEdit" name="tanggalDaftarDateEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="tanggalLoginLabel">
      <property name="text">
       <string>Tanggal Login</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateEdit" name="tanggalLoginDateEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="tanyaLabel">
      <property name="text">
       <string>Tanya</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="tanyaLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="jawabLabel">
      <property name="text">
       <string>Jawab</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="jawabLineEdit"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="idKotaLabel">
      <property name="text">
       <string>Id Kota</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="idKotaLineEdit"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="kotaPelangganLabel">
      <property name="text">
       <string>Kota pelanggan</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="kotaPelangganLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
